person.rb

```diff
@@ -1,5 +1,10 @@
-class Person
+require_relative './nameable'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
+
+class Person < Nameable
   def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
```

Comment:
**You did a great job!**

- [ ] However, in the Person class, I cannot find the `correct_name` method. If you want to correct names you need to include them in the Person class. Kindly add it.

Reply:
Implemented. Thank you for your helpful comment @Rustamxon7